process/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - process
 
+## 1.10.4 / 2019-12-23
```

Comment:
Hmm, `Added` should have triggered `1.11.0` upon doing `ddev release make process`